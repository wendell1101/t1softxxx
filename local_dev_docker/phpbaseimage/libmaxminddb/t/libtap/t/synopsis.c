#include "tap.h"

int main () {
    plan(5);
    ok(3 == 3);
    is("fnord", "eek", "two different strings not that way?");
    ok(3 <= 8732, "%d <= %d", 3, 8732);
    like("fnord", "f(yes|no)r*[a-f]$");
    cmp_ok(3, ">=", 10);
    done_testing();
}

