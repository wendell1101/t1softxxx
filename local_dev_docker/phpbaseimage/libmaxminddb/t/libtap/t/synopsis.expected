1..5
ok 1
not ok 2 - two different strings not that way?
#   Failed test 'two different strings not that way?'
#   at t/synopsis.c line 6.
#          got: 'fnord'
#     expected: 'eek'
ok 3 - 3 <= 8732
ok 4
not ok 5
#   Failed test at t/synopsis.c line 9.
#     3
#         >=
#     10
# Looks like you failed 2 tests of 5 run.
